src/components/task/TaskPanel.tsx

```diff
@@ -235,6 +269,44 @@ export class TaskPanel extends React.Component<Props, State> {
         break;
     }
   }
+
+  // stringifyNanoseconds takes an integer count of nanoseconds and
```

Comment:
Forgot to change this comment. Also could you add a couple of unit tests for this function?

Reply:
Done